dali/base/dafdesc.cpp

```diff
@@ -3651,12 +3651,17 @@ class CStoragePlaneInfo : public CInterfaceOf<IStoragePlane>
         Owned<IPropertyTreeIterator> srcAliases = xml->getElements("aliases");
         ForEach(*srcAliases)
             aliases.push_back(new CStoragePlaneAlias(&srcAliases->query()));
+        Owned<IPropertyTreeIterator> srcHosts = xml->getElements("hosts");
+        ForEach(*srcHosts)
+            hosts.push_back(std::string(srcHosts->query().queryProp(nullptr)));
```

Comment:
minor: I think the following works and is more efficient:

    hosts.emplace_back(srcHosts->query().queryProp(nullptr));

Reply:
changed (squahed in)